daemon/client/cli/cli.go

```diff
@@ -14,7 +14,7 @@ func (c *CliClient) Exec() error {
 	return c.rootCmd.Execute()
 }
 
-func (c *CliClient) Close(*cobra.Command, []string) error {
+func (c *CliClient) Close() error {
```

Comment:
You need to accept those args to match the type for PersistentPostRunE, but then it means you can't just Close it. Now that I needed to Close it somewhere else (integration_test.go), I changed this to the bare form, and then pass an anonymous func to PersistentPostRunE below, that just discards these args.